jobbergate-api/CHANGELOG.rst

```diff
@@ -6,6 +6,9 @@ This file keeps track of all notable changes to jobbergate-api
 
 Unreleased
 ----------
+
+3.3.4-alpha.0 -- 2022-11-30
+---------------------------
 - Added version metadata to the API
```

Comment:
I prefer to keep this as unreleased than fill out the CHANGELOG with lots of "unnecessary" versions. Then you can write what happened for that particular alpha version using the tag message. Thoughts?

Reply:
The procedure we are following in the project is to create new entries for each pre-release (it is actually automatic), so we can easily differentiate each of them. Once tested and ready for a stable release, we merge the pre-release entries under the stable version number, to reduce these unnecessary versions.